Makefile

```diff
@@ -88,7 +88,6 @@ test-node:
 
 .PHONY: test-browser
 test-browser:
-	cp node_modules/opentracing/dist/opentracing-browser.js test/dist
 	cp dist/lightstep-tracer.js test/dist
 	cd test && node ../node_modules/webpack/bin/webpack.js unittest_browser.js dist/unittest_browser.bundle.js
```

Comment:
I'm a bit confused here (might be because it's late) - if users don't have `node_modules` then how can this test suite run?

Reply:
They do have node_modules. But the opentracing-javascript package inside the node_modules folder no longer has build files in it (the build files are no lomnger checked into that project), and so copying opentracing-javascript build files from the package directory inside node_modules is no longer an option.

Reply:
ahhh ok. makes sense now